### ISSUE TITLE
Fill button doesn't work for furnaces

### ISSUE THREAD

**Issue by MeeniMc** [emilyploszaj/emi#83]
The 'fill' action doesn't work when interacting with a furnace, blast furnace, smokers.

## Replication Steps

1. Place a furnace
2. Put some fuel in the furnace
3. Get some cobblestones
4. Set mode to 'Craftable' (bottom right button in EMI interface)
5. Click the "Stone" on the right hand side list: **nothing happens**
6. Use the "r" keybind on the "Stone"
7. Click the "Fill Recipe button" (+ button in the center interface): **nothing happens**

## Versions

```
Mods:
  [✔] emi-0.4.0+1.19
  [✔] EMITrades-fabric-1.0.2+mc1.19.2
  [✔] fabric-api-0.61.0+1.19.2
  [✔] lithium-fabric-mc1.19.2-0.8.3
  [✔] modmenu-4.0.6
  [✔] reeses_sodium_options-1.4.7+mc1.19.2-build.59
  [✔] sodium-fabric-mc1.19.2-0.4.4+build.18
  [✔] starlight-1.1.1+fabric.ae22326
```

**Comment by emilyploszaj** [comment]
Can confirm, I noticed this the other day

**Comment by emilyploszaj** [comment]
Finally fixed in 0.4.3
